kie-spring-boot/kie-spring-boot-samples/jbpm-spring-boot-sample-basic/src/test/java/org/jbpm/springboot/samples/events/emitters/CountDownLatchEmitter.java

```diff
@@ -41,13 +42,13 @@ public CountDownLatch getCountDownLatch() {
     @Override
     public void deliver(Collection<InstanceView<?>> data) {
         log.info("deliver {}", data);
-        latch.countDown();
+        countDownIfTaskInstanceView(data);
     }
 
     @Override
     public void apply(Collection<InstanceView<?>> data) {
         log.info("apply {}", data);
-        latch.countDown();
+        countDownIfTaskInstanceView(data);
```

Comment:
shouldn't we a similar logic or approach to the `drop` method to keep this class consistent? wdyt?

https://github.com/kiegroup/droolsjbpm-integration/blob/d7ef89dc56984e471c5c19237d5391a5818244eb/kie-spring-boot/kie-spring-boot-samples/jbpm-spring-boot-sample-basic/src/test/java/org/jbpm/springboot/samples/events/emitters/CountDownLatchEmitter.java#L57

Reply:
makes sense, done